open-metadata-implementation/view-services/server-author-view/server-author-view-server/src/main/java/org/odpi/openmetadata/viewservices/serverauthor/handlers/ServerAuthorViewHandler.java

```diff
@@ -185,6 +186,10 @@ public ServerAuthorViewHandler(String localServerUserId, String metadataServerUR
                         // do not have a description of the server yet.
                         platform.addStoredServer(storedServer);
                     }
+                    //`/open-metadata/platform-services/users/${userId}/server-platform/registered-services/access-services`;
+                    MetadataAccessPointConfigurationClient metadataAccessPointConfigurationClient = new MetadataAccessPointConfigurationClient(userId, "",resourceEndpoint.getResourceRootURL());
```

Comment:
Strange to be only retrieving the access services.  Seems to be missing requests for retrieving other registered services needed for configuring other types of OMAG Server?

Reply:
Yes I agree - I started duplicating the capability of the existing code, I realised the PlatformServices Client that Dino uses was a better place to get this information